doc/mrac_siso_direct_lya_rule_statespace.rst

```diff
@@ -0,0 +1,15 @@
+Model-Reference Adaptive Control (MRAC) siso, direct lyapunov rule
```

Comment:
I would change "siso" to "SISO" (since it is an acronym) and capitalize Lyapunov (since it is a proper name).